emulsify.info.yml

```diff
@@ -6,7 +6,7 @@ core_version_requirement: ^9 || ^10
 
 dependencies:
   - drupal:components (^2.1)
```

Comment:
Safe to assume that this is D10 compatible too?

Reply:
Good call - changed that to `^3.0`